### PR TITLE
TropiCataclysm: More South American fauna + Old World newcomers

### PR DESCRIPTION
**Pull request by Xaritscin**
#### Summary
Mods "Adds some more SA mammals and a few imported from abroad"

#### Purpose of change

Mammal galore in mod was a bit rusty as i was focused on zeds, bugs and reptiles so i decided to give it a facelift with some more additions to further cement the Central-South American vibe of the mod along with a few animals escaped from different sources.

#### Describe the solution

-updated Peccaries
-added camels
-added llamas
-added spectacled bears (replaces the black bear)
-added bush dogs (kind of replaces coyotes but they are technically harmless to humans unless provoked)
-added capybaras 
-added wallabies (bit of a homage to the Hook Kangaroo from Desert Region but fairly smaller and unzombified)

all the aforementioned animals were added to their respective monstergroups, including a new misc.json to see if camels, llamas, guinea pigs, guinea fowl, quails and geese can spawn on farms now.

#### Describe alternatives you've considered

We may have a lot of mammals on mod already.

#### Testing

Loaded my debug character on a fresh map, jumped around from farm to farm looking at barns. found 2 barns with guinea fowl and one with some llamas and a camel so they are working as intended on their respective monstergroup.

River spawn had capybaras among other common sights like geese and turtles. No idea regarding forest mammals but going by weights bush dogs, bears and wallabies should spawn around on their respective places and chances on overmap.

#### Additional context

Was planning to add more bats along with sloths, armadillos and anteaters but i didnt want to clog the PR too much. maybe on future updates.

### CODE REVIEW

File: data/mods/TropiCataclysm/monsters/tropical_mammal.json [hunk 1]
```diff
@@ -124,6 +140,66 @@
     "special_attacks": [ [ "EAT_FOOD", 20 ] ],
     "flags": [ "SEES", "HEARS", "SMELLS", "PET_MOUNTABLE", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM", "KEENNOSE", "SWIMS" ]
   },
+  {
+    "id": "mon_camel",
+    "type": "MONSTER",
+    "name": { "str": "camel" },
+    "description": "A large herbivore introduced from the old world via zoos and private owners.  Ironically, camels were once native to the american continent in distant times.",
```
Comment (NetSysFire) [inline data/mods/TropiCataclysm/monsters/tropical_mammal.json:147]:
```suggestion
    "description": "A large herbivore introduced from the old world via zoos and private owners.  Ironically, camels were once native to the American continent in distant times.",
```

Just American continent sounds wird. Maybe specify if its North or South America or just "native to America".

File: data/mods/TropiCataclysm/monsters/tropical_mammal.json [hunk 2]
```diff
@@ -124,6 +140,66 @@
...
+    "biosignature": { "biosig_item": "feces_manure", "biosig_timer": 2 },
+    "special_attacks": [ [ "EAT_CROP", 60 ] ],
+    "petfood": { "food": [ "CATTLEFOOD" ], "feed": "The %s seems to like you!  It lets you pat its head and seems friendly." },
+    "flags": [ "SEES", "HEARS", "SMELLS", "ANIMAL", "PET_WONT_FOLLOW", "PET_MOUNTABLE", "PATH_AVOID_DANGER_1", "WARM" ]
+  },
+  {
+    "id": "mon_bushdog",
+    "type": "MONSTER",
+    "name": { "str": "bush dog" },
+    "description": "A small and elusive canid native to South and Central America.  Its harmless to humans but their packs can take on prey several times their size.",
```
Comment (NetSysFire) [inline data/mods/TropiCataclysm/monsters/tropical_mammal.json:179]:
```suggestion
    "description": "A small and elusive canid native to South and Central America.  It is harmless to humans but their packs can take on prey several times their size.",
```

File: data/mods/TropiCataclysm/monsters/tropical_mammal.json [hunk 3]
```diff
@@ -179,11 +289,77 @@
...
+    "upgrades": { "age_grow": 240, "into": "mon_llama" },
+    "//": "Puberty reached in 6-9 months, copied from lamb but it probably applies the same on llamas.",
+    "biosignature": { "biosig_item": "feces_manure", "biosig_timer": 7 },
+    "flags": [ "SEES", "HEARS", "SMELLS", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM" ]
+  },
+  {
+    "id": "mon_llama",
+    "type": "MONSTER",
+    "name": { "str_sp": "llama" },
+    "description": "A fairly large and wooly herbivore endemic to the American Continent.  Llamas and their kin are distant relatives to camels and were domesticated by the andean people.",
```
Comment (NetSysFire) [inline data/mods/TropiCataclysm/monsters/tropical_mammal.json:332]:
```suggestion
    "description": "A fairly large and wooly herbivore endemic to the American Continent.  Llamas and their kin are distant relatives to camels and were domesticated by the Andean people.",
```

**Comment by Xaritscin** [comment]
Mod was working with no crashes whilte testing. seems ready for review

**Pull request merged**